plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1291,8 +1290,8 @@ public void testInsertBatchSizeSessionProperty(Integer batchSize, Integer number
         return result;
     }
 
-    @DataProvider(name = "testInsertBatchSizeSessionProperty")
-    public static Object[][] batchSizeAndNumberOfRowsForInsertBatchSizePropertyTest()
+    @DataProvider
+    public static Object[][] batchSizeAndTotalNumberOfRowsToInsertDataProvider()
```

Comment:
nit: I think I like the existing convention of test method name + DataProvider but this works too.

`testInsertBatchSizeSessionPropertyDataProvider`

Reply:
Agree. I like it too, but I think in this case `batchSizeAndTotalNumberOfRowsToInsertDataProvider` is more descriptive.

Reply:
Agreed.